herdimmunity/src/main/scala/com/revature/scalawags/project3/herdimmunity/CSVPullerAndParser.scala

```diff
@@ -33,25 +32,27 @@ object CSVPullerAndParser  extends App{
                 
                 var peopleVaccinated = splitLine(35)
                 if (peopleVaccinated == ""){peopleVaccinated = "0.0"}
-                var peopleVaccinatedToDouble = peopleVaccinated.toDouble
+                var peopleVaccinatedDouble = peopleVaccinated.toDouble
```

Comment:
Does casting peopleVaccinated directly to Int (instead of first to Double then to Int) not work here?

Reply:
No it doesn't work unless you cast to double then to int